tests/utils.py

```diff
@@ -76,6 +76,12 @@
 IS_NOT_ON_GITHUB = os.getenv("GITHUB_ACTIONS") != "true"
 # this env variable is set in regression test
 IS_IN_STORED_PROC_LOCALFS = IS_IN_STORED_PROC and os.getenv("IS_LOCAL_FS")
+STRUCTURED_TYPE_ENVIRONMENTS = {"dev", "aws"}
+IS_STRUCTURED_TYPES_SUPPORTED = (
+    os.getenv("cloud_provider", "dev") in STRUCTURED_TYPE_ENVIRONMENTS
```

Comment:
cloud_provider is set on github, right? We also have some tests running on jenkins, so they will just be skipped on jenkins?

Reply:
I'm not sure how that gets set. I know that in tox.ini it's passed in regardless of environment, but that doesn't mean it is set.

Reply:
so we will skip these tests on jenkins right now? e.g., https://ci-dev-142.int.snowflakecomputing.com/job/SnowparkPythonClientRegressRunner/

Reply:
Yes. Once the parameter for enabling structured types is on by default we will need to make sure these tests work in all environments, but for now sfctest0 is the only environment that has them enabled. It is also the only environment that we have iceberg test infrastructure set up.

Reply:
got it, thanks. Then maybe create a jira and add a TODO here to make sure these conditions will be finally removed?